lib/players/human.rb

```diff
@@ -20,6 +21,10 @@ def get_move(grid, turns_remaining)
     return move
   end
 
+  def exit?(move_string)
+    move_string.to_sym == :exit
+  end
+
```

Comment:
Something doesn't feel quite right about having this here, but I can't put my finger on it.

Somewhat related, have you examined this classes responsibilities lately? Think about if there is an opportunity to refactor and simplify this class in the future.

Reply:
Yeah, exit? was included in the human class for now.  I have reworked the human class on [another branch](https://github.com/saralein/tic-tac-toe_ruby/pull/14/files#diff-30df6bc95cd728536b006cd643f0e0caR1) so that validation is no longer there and the class just gets moves.  My plan was to remove this in the future merge.